cypress/integration/taskCloneTableColumn.spec.js

```diff
@@ -1,33 +1,24 @@
 'use strict';
 
 describe('Clone table app', () => {
-  before(() => {
+  beforeEach(() => {
     cy.visit('/');
+    cy.get('th:nth-child(2)').debug();
   });
 
   it('should only change number of columns in original table', () => {
     cy.get('table').children().should('have.length', 3);
-    cy.get('thead > tr').children().should('have.length', 6);
+    cy.get('thead > tr > th').should('have.length', 6);
     cy.get('thead').children().should('have.length', 1);
     cy.get('tbody').children().should('have.length', 12);
```

Comment:
Similar to the previous comment, asserting the exact number of children for `tbody` could make your test brittle if the data changes. If you need to ensure a minimum set of rows, consider checking for at least that number instead of an exact match.